docs/install_guides/pypi-openvino-dev.md

```diff
@@ -6,15 +6,7 @@ OpenVINO™ Development Tools enables you to download models from Open Model Zoo
 
 ## System Requirements
 
-Before you start the installation, check the supported operating systems and required Python* versions. The complete list of supported hardware is available in the [Release Notes](https://www.intel.com/content/www/us/en/developer/articles/release-notes/openvino-relnotes.html).
-
-| Supported Operating System                                   | [Python* Version (64-bit)](https://www.python.org/) |
-| :------------------------------------------------------------| :---------------------------------------------------|
-|   Ubuntu* 18.04 long-term support (LTS), 64-bit              | 3.7, 3.8, 3.9, 3.10                                 |
-|   Ubuntu* 20.04 long-term support (LTS), 64-bit              | 3.7, 3.8, 3.9, 3.10                                 |
-|   Red Hat* Enterprise Linux* 8, 64-bit                       | 3.7, 3.8, 3.9, 3.10                                 |
-|   macOS* 10.15.x                                             | 3.7, 3.8, 3.9, 3.10                                 |
-|   Windows 10*, 64-bit                                        | 3.7, 3.8, 3.9, 3.10                                 |
+Before you start the installation, check the supported operating systems and required Python* versions. The complete list of supported hardware is available in the [System Requirements](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/system-requirements.html).
```

Comment:
The `pypi-openvino-dev.md` and `pypi-openvino-rt.md` files are for pypi.org. I'm good with pointing users to the single-sourced System Requirements page. But I guess decisions should be made by @ilya-lavrenov .

Reply:
I personally OK for a single place, but I believe this place is OpenVINO documentation, but external site like https://www.intel.com/content
I don't like that users need to jump from one resource to another

Reply:
Yes, I totally agree that system requirements belong to software documentation and should be kept in the documentation page. For now, there's nothing we can do about it but I'll keep lobbying for this change.